docker/domserver/scripts/start.d/50-domjudge.sh

```diff
@@ -87,6 +89,32 @@ else
 	echo "TRUSTED_PROXIES=${DOCKER_GATEWAY_IP}" >> webapp/.env.local
 fi
 
+# Add custom trusted proxies
+if [ -n "${TRUSTED_PROXIES}" ]; then
+	PRE_TRUSTED_PROXIES=$(grep "TRUSTED_PROXIES" webapp/.env.local | sed 's|TRUSTED_PROXIES=||g')
```

Comment:
Did you now remove the wrong one, i.e. docker gateway should stay but pre trusted proxies not, right?